packages/flutter_tools/lib/src/doctor_validator.dart

```diff
@@ -190,20 +190,7 @@ class ValidationResult {
   }
 
   /// The string representation of the type.
-  String get typeStr {
-    switch (type) {
-      case ValidationType.crash:
-        return 'crash';
-      case ValidationType.missing:
-        return 'missing';
-      case ValidationType.success:
-        return 'installed';
-      case ValidationType.notAvailable:
-        return 'notAvailable';
-      case ValidationType.partial:
-        return 'partial';
-    }
-  }
+  String get typeStr => type.name;
```

Comment:
is this not a breaking change for the GA3 events?

Reply:
Not necessarily, the events we send for `ValidationType.success` currently have the value `installed`. Any dashboards/queries that are looking for successes will need to be updated to look for both `installed` and `success` values and they will be capturing the same information.

Although I didn't see any dashboards relating to `flutter doctor` events in go/flutter-dash... @jayoung-lee can you confirm if we are doing any reporting on doctor events?

Reply:
I know Tao has queried this before for a study. How would someone in the future querying this field know they have to sum "installed" **and** "success"?

Reply:
I'm not sure if we currently have a place we keep this kind of information for GA3, but GA4, I can work with the UXR team to ensure we have a live document that keeps track of these changes

Reply:
I think for now we should focus on the minimal changes needed to get event parity to GA4.

Reply:
Sure, done